docs/HowTo/Deploy/Monitoring-Performance.md

```diff
@@ -71,9 +71,23 @@ block of the `prometheus.yml` file:
 1. Choose **Graph** from the menu bar and click the **Console** tab below.
 
 1. From the **Insert metric at cursor** drop-down, select a metric such as `besu_blockchain_difficulty_total` or
-`besu_blockchain_height` and click **Execute**. The values are displayed below.
+`besu_blockchain_height` and click **Execute**. The values are displayed.
```

Comment:
besu_blockchain_height has been replaced by ethereum_blockchain_height (that's the change the PR linked in the EIP made)